server/auth/login.py

```diff
@@ -62,9 +73,9 @@ def login_request(self, packet, address):
         permission_error = 'You do not have permission to log in to this server' if auth_status == 'not_permitted' else ''
 
         if uid:
-            self.server.handle_until_value('auth:set_address', uid, address)
-            new_subscriber = self.server.handle_until_value('auth:new_subscriber', True, uid)
-            ftp = self.server.handle_until_value('auth:free_to_play', True, uid)
+            self.server.handle_until_value('auth:set_address', True, uid, address)
+            new_subscriber = self.server.handle_until_return('auth:new_subscriber', uid)
```

Comment:
These either need to be true or false, so handle_until_value would be the better method

Reply:
Doesn't this return a bool already tho?

Reply:
Yes, it does. However, the intention of `handle_until_value` is that a specific value is expected to be returned (which is the case here), as opposed to `handle_until_return`, which would be used when the value to return is unknown

Reply:
Ah, I see